docs/static/js/version.js

```diff
@@ -0,0 +1,111 @@
+// Uncomment the following for debug
+// READTHEDOCS_DATA = {
+//     "ad_free": false,
+//     "api_host": "https://readthedocs.org",
+//     "build_date": "2022-01-25T06:27:55Z",
+//     "builder": "sphinx",
+//     "canonical_url": null,
+//     "commit": "ca66e346",
+//     "docroot": "/docs/en_US/",
+//     "features": { "docsearch_disabled": false },
+//     "global_analytics_code": "UA-17997319-1",
+//     "language": "en",
+//     "page": "Tutorial",
+//     "programming_language": "words",
+//     "project": "nni",
+//     "proxied_api_host": "/_",
+//     "source_suffix": ".rst",
+//     "subprojects": { "nni-zh": "https://nni.readthedocs.io/zh/stable/" },
+//     "theme": "sphinx_material",
+//     "user_analytics_code": "UA-136029994-1",
+//     "version": "latest"
+// };
+
+// READTHEDOCS_VERSIONS = [
+//     ["latest", "/en/latest/"],
+//     ["stable", "/en/stable/"],
+//     ["v2.6", "/en/v2.6/"],
+//     ["v2.5", "/en/v2.5/"],
+//     ["v2.4", "/en/v2.4/"],
+//     ["v2.3", "/en/v2.3/"]
+// ];
+// The above code is injected by readthedocs in production.
+
+
+function create_dropdown(button_text, items) {
+    var dropdown = document.createElement("div");
+    dropdown.className = "md-flex__cell md-flex__cell--shrink drop";
+    var button = document.createElement("button");
+    button.innerHTML = button_text;
+    var content = document.createElement("ul");
```

Comment:
maybe use `const` is better, and line 36, 38, 40 should be put together for better performance

Reply:
I'm writing ES5 because I thought `const` are not supported by some browsers. Figured that I'm outdated.